book/eha.qmd

```diff
@@ -81,17 +96,73 @@ $$\delta_{ie} = \II(Y_i \leq C_i \wedge E_i = e) \in \{0, 1\},\ e=1,\ldots,k$$
+
+When modeling such data, interest often lies in estimation of the transition hazards, but also in transition probabilities $P_{\ell e}(\zeta, \tau)$, that is the probability to transition from state $\ell$ to $e$ between time points $\zeta$ and $\tau$ or state occupation probabilities $P_e(\tau)$, the probability to occupy state $e$ at time $\tau$.
+Transition probabilities are often summarized in a matrix of transtion probabilities $\mathbf{P}(\zeta, \tau)$, where rows indicate starting states and columns indicate end states.
+
+For example, for the illness-death model, the transition matrix would be given as
+
+$$\mathbf{P}(\zeta, \tau) :=
+  \begin{pmatrix}
+  P_{00}(\zeta, \tau) & P_{01}(\zeta, \tau) & P_{02}(\zeta, \tau)\\
+  \bullet & P_{11}(\zeta, \tau) & P_{12}(\zeta, \tau)\\
+  \bullet & \bullet & P_{22}(\zeta, \tau)\\
+  \end{pmatrix}.
+$$
+
+$\bullet$ indicates impossible transitions, as in the illness-death model there are no transitions from higher to lower states.
+
+
+
+## Recurrent Events {#sec-recurrent-events}
 
-![Illustration of transitions in the recurrent events setting. Left: "Clock reset" representation of recurrent events. Each time an event occurs, the clock is reset to 0 and we once again wait for occurence of an event (of the same type); Right: "Clock forward" representation. A subject transitions from 1-st, to 2-nd, etc. to K-th event occurence](Figures/survival/recurrent-events.png){#fig-recurrent-events fig-alt="Schematic illustration of recurrent events transitions."}
+The *recurrent events* setting is defined by the same event type occurring multiple times.
+While the recurrent events setting can be viewed as a special case of the multi-state setting (@sec-multi-state), we treat it here separately as it can also be approached as single event data with correlation.
+Some examples of recurrent events include recurrence of infectious diseases, episodes of epilepsy, and replacements of machine parts.
+
+@tbl-cgd-data shows data of chronic granolotomous disease (CGD) for two subjects.
+Note the start-stop notation of the data ($t_{start}, t_{stop}$ respectively) that indicates the time in which a subject enters the risk set for the k*th* event number.
+The column $\delta$ indicates whether the subject experienced the $k$th outbreak or was censored.
+For example, Subject 1 (first three rows) experienced the first outbreak (first row) after 219 days, at which time they entered the risk set for event number 2.
+The second event (second row) was experienced at 373 days and the subject was then censored for the third event (third row) after 414 days.
+This notation is equivalent to a multi-state representation of the recurrent events process, where each event number represents a different state.
+In @fig-recurrent-events, right panel, this is refered to as "Clock forward" approach.
```

Comment:
Small point but in general I think when there's multiple plots in a figure they should be presented in order, i.e. the clock-forward plot should be on the left

Reply:
Also why is it called clock forward?